parser/routing.go

```diff
@@ -0,0 +1,97 @@
+package parser
+
+import (
+	"github.com/deis/workflow-cli/cmd"
+	docopt "github.com/docopt/docopt-go"
+)
+
+// Routing displays all relevant commands for `deis routing`.
+func Routing(argv []string) error {
+	usage := `
+Valid commands for routing:
+
+routing:list       list routability for an app
```

Comment:
I think just `apps:info` would be fine, but if I was going to look up routability and had no idea how, I would run `deis routing help` and would expect some singular command like `info` and would be confused if `routing:list` was what I was looking for.